library/core/src/num/uint_macros.rs

```diff
@@ -2603,8 +2610,16 @@ macro_rules! uint_impl {
         /// # Examples
         ///
         /// ```
-        #[doc = concat!("assert_eq!(1", stringify!($SelfT), ".wrapping_shl(7), 128);")]
-        #[doc = concat!("assert_eq!(1", stringify!($SelfT), ".wrapping_shl(128), 1);")]
+        #[doc = concat!("assert_eq!(1_", stringify!($SelfT), ".wrapping_shl(7), 128);")]
+        #[doc = concat!("assert_eq!(42_", stringify!($SelfT), ".wrapping_shl(", stringify!($SelfT), "::BITS), 42);")]
+        /// assert_eq!(
+        #[doc = concat!("    42_", stringify!($SelfT))]
+        #[doc = concat!("        .wrapping_shl(", stringify!($SelfT), "::BITS / 2)")]
+        #[doc = concat!("        .wrapping_shl(", stringify!($SelfT), "::BITS / 2),")]
+        ///     0,
+        /// );
```

Comment:
I wrapped it this way because that's what rustfmt does, apparently: <https://play.rust-lang.org/?version=stable&mode=debug&edition=2024&gist=a2b9ff9292bc0f28534eb6e780b14ff8>